src/components/elements/CommonTabs.tsx

```diff
@@ -0,0 +1,71 @@
+import { Box, SxProps, Tab, Tabs } from '@mui/material';
+import React, { ReactNode, useState } from 'react';
+
+export type TabDefinition = {
+  title: string;
+  contents: ReactNode;
+};
+
+interface CommonTabsProps {
+  tabDefinitions: TabDefinition[];
+  ariaLabel: string;
+  sx?: SxProps;
+  currentTab?: number;
+  onChangeTab?: (tab: number) => void;
+}
+
+// CommonTabs wraps the MUI Tabs component. It can be controlled or uncontrolled,
+// so callers can choose whether to delegate controlling the state of which tab is selected.
+// This is a rare case where ariaLabel is a required prop, otherwise the tabs won't be accessible.
+const CommonTabs: React.FC<CommonTabsProps> = ({
```

Comment:
I didn't think to spec this before, but I do think we will want to adjust this at some point so it can support using hash params so that you can navigate directly to a tab (and have the current tab reflected in the URL), like `/projects/933/opportunities/4#overview` vs `/projects/933/opportunities/4#clients`. We have a implementation of that in the household assessments tab (maybe weird implementation, shouldn't necessarily copy).

Especially looking at the "referral queue" page which _also_ consists of tabbed tables, I think we will want this. But happy to defer to a separate ticket! (maybe it can be done as part of https://github.com/open-path/Green-River/issues/7350)

Reply:
Yeah I was thinking this too. I like the idea of including it in 7350's scope. I'll make a note on that ticket.

Reply:
Oh, this is a lot easier than I thought; we can just use `href`. No need for a fancy `useHashParamsState` hook or anything like that.

I think in this case I will take the time to split this PR out into 2 PRs, one with the CommonTabs implementation and the other with the Opportunity Page changes.